shared/data/build.gradle.kts

```diff
@@ -51,7 +51,8 @@ apollo {
 
     introspection {
       schemaFile.set(file("src/commonMain/graphql/schema.graphqls"))
-      endpointUrl.set("https://androidmakers-2023.ew.r.appspot.com/graphql")
+      endpointUrl.set("https://confetti-app.dev/graphql")
+      headers.set(mapOf("conference" to "androidmakers2024"))
```

Comment:
Is this required for introspection? I would hope the schema does not depend on the conference

Reply:
No but the IJ plugin also looks at this headers and inject them when running queries from the UI :)

Reply:
I see. Well this is a bit surprising. Might adding a comment?